docs/how-to-deploy-cp4waiops-customsize.md

```diff
@@ -0,0 +1,353 @@
+   - "Username": `cp`
+   - "Password": paste the entitlement key you copied from the [Obtain an entitlement key](#obtain-an-entitlement-key) setp
+   - "Email": any email, valid or not, will work. This fields is mostly a hint to other people who may see the entry in the configuration
+
+1. Click on "Save"
+
+
+
+## Install custom sizing CP4WAIOps from UI
+
+### Login to Argo CD
+
+You can now login to Argo CD UI as follows by clicking the drop down menu on top right.
+
+![w](images/gitops-menu.png)
+
+Argo CD UI will be popped up and you can login using `LOG IN VIA OPENSHIFT`.
+
+![w](images/gitops-login.png)
+
+
+### Grant Argo CD Cluster Admin Permission
+
+From Red Hat OpenShift Console, go to `User Management` > `RoleBindings` > `Create binding`. Use the form view to configure the properties for the `ClusterRoleBinding` with values as follows, and click the `Create` button.
+
+- Binding type
+  - Cluster-wide role binding (ClusterRoleBinding)
+- RoleBinding
+  - Name: argocd-admin
+- Role
+  - Role Name: cluster-admin
+- Subject
+  - ServiceAccount: check it
+  - Subject namespace: openshift-gitops
+  - Subject name: openshift-gitops-argocd-application-controller
+
+
+
+
+### Install custom sizing CP4WAIOps Using All-in-One Configuration
+
+
+#### Install AI Manager and Event Manager in One Go
+
+The all-in-one configuration allows you to install following components in one go:
+
+- Ceph storage (optional)
+- AI Manager
+- Event Manager
+
+For AI Manager, you can specify the installation profile or sub-profile of small with custom sizing, including:
+
+- large
+- small
+- extra small: sub-profile requesting much less cpu and memory, only for demo, PoC, or dev environment in a resource (cpu and memory) limited cluster.
+  - x-small
+  - x-small-idle
+  - x-small-custom
+  
+ And support SNO (Single Nodw OCP).  	 
+
+Just fill in the form using the suggested field values listed in following table when you create the Argo CD App:
+
+| Field                 | Value                                                 |
+| --------------------- | ----------------------------------------------------- |
+| Application Name      | anyname (e.g. cp4waiops-app)                          |
+| Project               | default                                               |
+| Sync Policy           | Automatic                                             |
+| Repository URL        | https://github.com/IBM/cp4waiops-gitops               |
+| Revision              | release-3.4                                           |
+| Path                  | config/all-in-one                                     |
+| Cluster URL           | https://kubernetes.default.svc                        |
+| Namespace             | openshift-gitops                                      |
+
+In Helm PARAMETERS: specify/override which profile to install:
+
+| Field                 | Value                                                        | Description
```

Comment:
Suggest to use the below table headline consistently for Helm PARAMETERS description.

| Parameter                | Type   | Default Value      | Description 
| ------------------- |--------|--------------------|-------------
| `<Parameter>`          | `<Type>` | `<Default Value>`      | `<Description>`

Reply:
Agree.